contrib/release/pull-docker-manifests.sh

```diff
@@ -3,21 +3,25 @@
 # Copyright 2020 Authors of Cilium
 
 DIR=$(dirname $(readlink -ne $BASH_SOURCE))
-source $DIR/lib/common.sh
+source "${DIR}/lib/common.sh"
 
 CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}
-IMAGES=(cilium clustermesh-apiserver docker-plugin hubble-relay operator operator-generic operator-aws operator-azure)
-REGISTRIES=(docker.io quay.io)
+
+repo="cilium/cilium"
 
 usage() {
-    logecho "usage: $0 <VERSION>"
-    logecho "VERSION    Target version"
+    logecho "usage: $0 <GH-USERNAME> <VERSION> <RUN-URL>"
+    logecho "GH-USERNAME  GitHub username"
+    logecho "VERSION      Target version"
+    logecho "RUN-URL      GitHub URL with the RUN for the release images"
```

Comment:
I think what you mean here is `RUN-ID` rather than the full URL, just looking at how the variable is used. It would not work if the full URL was passed. Also, I think we can be clearer on which job specifically has this ID. Could you put the job name in the help string so that it is unambiguous?

Reply:
I saw in later commits that we are supposed to save the URL from the GH run, but I think it would still help to put slightly more clarification.

Reply:
No, I meant `RUN-URL`, see line 69 where the `run_id` is derived from the url

Reply:
TIL. I didn't know `basename` can work with URLs :exploding_head: